website/src/pages/ga4gh/drs/v1/service-info.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import type { APIRoute } from 'astro';
+
+import { getWebsiteConfig } from '../../../../config';
+
+/**
+ * DRS Service Info type definition according to GA4GH Service Info v1.x
+ */
+interface DrsServiceInfo {
+    id: string;
+    name: string;
+    description: string;
+    organization: {
+        name: string;
+        url: string;
+    };
+    contactUrl: string;
+    documentationUrl: string;
+    createdAt: string;
+    updatedAt: string;
+    environment: string;
+    version: string;
+    type: {
+        group: string;
+        artifact: string;
+        version: string;
+    };
+}
```

Comment:
When I look at the docs I see these additional fields - https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.5.0/docs/#tag/Service-Info/operation/GetServiceInfo

```
  "maxBulkRequestLength": 0,
  "drs": {
    "maxBulkRequestLength": 0,
    "objectCount": 0,
    "totalObjectSize": 0
  }
```

Reply:
It seems like some of these aren't used a lot: https://github.com/search?q=maxBulkRequestLength&type=code maybe this whole endpoint isn't used so much?